binding/lua/init.lua

```diff
@@ -14,7 +14,16 @@ ffi.cdef[[
     int MV_ServerId();
 ]]
 
-libmv = ffi.load('libmultiverso', 'true')
+package.cpath = '/usr/local/lib/?.so;' .. package.cpath
+libmv_path = package.searchpath('libmultiverso', package.cpath, '')
+if libmv_path == nil then
+    print([[
+[Error] Multiverso shared object, `libmultiverso.so`, NOT FOUND!
+Please build & install `multiverso` according to the instruction [1].
+[1] https://github.com/Microsoft/multiverso#build]])
```

Comment:
Actually `package.searchpath()` takes a list of candidate paths as parameter for searching. That's why I choose to be not specified. And there may be something I should clarify. We will change the build process of multiverso and install the shared object in proper place, so the user don't need to care about the exact place it should be. As the url will be the same one, I think it's ok to leave a link here for reference as we will change the content soon.